lambda_functions/automate_s2_nbar_rolling_archive/execute_s2nbar/handler.py

```diff
@@ -0,0 +1,36 @@
+import boto3
+import logging
+import os
+from ssh import exec_command
+
+LOG = logging.getLogger(__name__)
+LOG.setLevel(logging.INFO)
+
+
+def automate_rolling_archive(s3_client, s3_bucket, filelist, execute_command=exec_command):
+    execute_command(f'execute_s2nbar_rolling_archive --s3client {s3_client} --s3bucket {s3_bucket} '
```

Comment:
Have you tested this? I'm 99% sure it's not going to work at all.

You're creating a boto3 client in the lambda, and then passing it as a command line string to run on `raijin`. `{s3_client}` is going to look something like `'<botocore.client.S3 object at 0x10b70b320>'`.

Reply:
I have updated handler file to do it in a different way. I am not passing boto client object as a command line string to run on raijin.